libs/game/scene.ts

```diff
@@ -23,6 +23,11 @@ namespace scene {
         handler: (sprite: Sprite) => void
     }
 
+    export interface GameForeverHandlers {
+        active: boolean;
```

Comment:
Nitpick: Maybe "lock" or "running" instead of active? Active makes it sound like it has been enabled/disabled or something.